specs/pvs/corla.pvs

```diff
@@ -2056,9 +2055,15 @@ BEGIN
     [# location: string, date: string #]
   rla_information_table: TYPE =
     sequence[[RLA, audit_reason, contest, risk_limit]]
-% @todo nealmcb rename to audited_ballot_list_table?
-  ballots_to_audit_list_table: TYPE =
+%%% We formalize the list of ballots under audit and organize them
+%%% both by contest and by county. The former is defined when the RLA
+%%% algorithm randomly chooses ballots. The latter is populated by
+%%% filtering the former by county, and it is what is shared with
+%%% counties via the Audit Board Dashboard.
+  audited_ballot_by_contest_list_table: TYPE =
     sequence[[contest, sequence[ballot_manifest_info]]]
```

Comment:
And we also have to choose by county, not contest, so we end up selecting the same ballots for each contest.
Proposed text:
```
%%% We formalize the list of ballots under audit and organize them
%%% both by county and by contest. The former is defined when the RLA
%%% algorithm randomly chooses ballots, and it is what is shared with
%%% counties via the Audit Board Dashboard. The latter is populated by
%%% filtering the former by contest, for future use.
```
and then swap the order of the definitions.